appveyor.yml

```diff
@@ -14,26 +14,24 @@ environment:
     PYTHONUNBUFFERED: 1
 
   matrix:
-    - PYTHON: "C:\\Miniconda"
-      CONDA_PY: "27"
-      ARCH: '32'
-    - PYTHON: "C:\\Miniconda-x64"
-      CONDA_PY: "27"
-      ARCH: '64'
-    - PYTHON: "C:\\Miniconda36"
-      CONDA_PY: "36" # No MDTraj for py37/Win32
-      ARCH: '32'
+    #- PYTHON: "C:\\Miniconda38-x64"
+      #CONDA_PY: "38" 
+      #ARCH: '64'
     - PYTHON: "C:\\Miniconda37-x64"
       CONDA_PY: "37"
       ARCH: '64'
+    - PYTHON: "C:\\Miniconda36-x64"
+      CONDA_PY: "36" 
+      ARCH: '64'
 
 build: false
 
 install:
   # install python
-  - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%
+  - SET PATH=%PYTHON%;%PYTHON%\\Scripts;%PYTHON\\bin;%PATH\\Library\\bin;%PATH%
```

Comment:
Why are the extra PATH variables needed? (just asking for info)

Reply:
removed; but may be required in the future. Newer versions of conda have moved the conda executable around, so if we're manually setting `PATH`, it might have moved. I added these while searching for it in the supposed-to-exist Miniconda38 on AppVeyor.